app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -322,6 +322,27 @@ class NavigationDrawerTestActivityTest {
     }
   }
 
+  @Test
+  // TODO(#973): Fix NavigationDrawerTestActivityTest
```

Comment:
Sorry to be clear, we should be checking in a test that works here. The other tests can be fixed in #973, but this one needs to be working (we shouldn't be checking in disabled tests anymore).

Reply:
done